test/support/generators.ex

```diff
@@ -0,0 +1,98 @@
+defmodule SchematicTest.Generators do
+  @moduledoc """
+  Generators for property-based testing
+  """
+
+  import Schematic
+
+  def json_primitive() do
```

Comment:
let's not do the json related thing.

schematic can be used for validating external json data, but it can also be used from elixir->elixir data structures, which means it should be tested with all beam terms.